packages/tools/ds-version-history/yarn.lock

```diff
@@ -0,0 +1,3950 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
you don't need this file. aren't things automatically installing once you run yarn?

Reply:
This was auto-generated when I ran yarn. Also, if we don't need this file, we can add it to `gitignore`.